### PR TITLE
Upgrade node-sass version to meet USWDS requirements

### PR DESCRIPTION
**Pull request by cehsu**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1463

No errors encountered when upgrading.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5866,7 +5866,25 @@ node-pre-gyp@^0.6.29, node-pre-gyp@~0.6.32:
     tar "^2.2.1"
     tar-pack "^3.4.0"
 
-node-sass@^3.10.1, node-sass@^3.4:
+node-sass@3.4.2:
```
Comment (jbalboni) [inline yarn.lock]: The newest version of node-sass is 4.5.3. That version should get installed if you run `yarn upgrade node-sass`.

Comment (cehsu) [inline yarn.lock]: It looks like USWDS requires "node-sass": "^3.4.2", but we are currently requiring "^3.10.1", and installing "3.13.1". This PR requires the version of node-sass that USWDS requires. Is it better to simply upgrade to the latest version?

Comment (jbalboni) [inline yarn.lock]: Ah, I see. No, let's not get out of sync with uswds. This version is fine.

**Review by jbalboni — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]
If we're going from 3.10.1 -> 3.4.2, it's not really an upgrade, is it? 😞  Still, best to not get out of sync just in case, eh?

**Pull request merged**